packages/material-ui/src/styles/createTransitions.js

```diff
@@ -43,7 +43,7 @@ function getAutoHeightDuration(height) {
   return Math.round((4 + 15 * constant ** 0.25 + constant / 5) * 10);
 }
 
-export default function createTransitions(inputTransitions) {
+export default function createTransitions(inputTransitions = {}) {
```

Comment:
Let's not change the runtime behavior.

Reply:
I added it because the implementation spread the value and it looks fine to have default value as `{}`. Any concern? or you don't want to change it at this point?

Reply:
I just don't want to change anything without cause.